packages/klevu-ui/src/components/klevu-chat-messages/klevu-chat-messages.tsx

```diff
@@ -71,6 +80,36 @@ export class KlevuChatMessages {
 
   feedbackReasons = ["Irrelevant", "Incorrect", "Offensive", "Other"]
 
+  @State() lastMessageDisplayedText: string = '';
+  @State() typeWriterEnds: boolean = true;
+  
+  @Watch('messages')
+
+  watchPropHandler(newValue: any, oldValue: MoiMessages) {
+    const lastItem = [...newValue].pop() || null; ;
```

Comment:
is it better to just use last index? rather than creating new array object

Reply:
changed to at()